tests/unit/test_instance_cmds.py

```diff
@@ -1564,22 +1560,24 @@
       'test': 'in'},
      ASSOC_MOCK_FILE, OK],
 
+    # Behavior changed pywbem 0.15.0 to exception rtn
     ['Verify instance command references, invalid instance name',
      ['references', 'TST_Blah.blah="abc"'],
-     {'stdout': "",
-      'rc': 0,
-      'test': 'lines'},
+     {'stderr': "",
+      'rc': 1,
+      'test': 'innows'},
      ASSOC_MOCK_FILE, OK],
 
+    # Because the order of pick is not guaranteed, we test minimum data
+    # TODO: Marked fail because for some rason with pywbem 0.15.0, this
+    # test fails without building instance sometimes.
```

Comment:
I suggest to investigate this until it is understood what the circumstances are, and how to specify the testcases as a result.

Reply:
I want to get the majority of this PR in since without it we cannot pass tests since we move to 0.15.0. That is why I marked this as TODO since it is going to take time to sort it out.  This affects two tests that mock the prompt for the select instance names method. I would rather mark this as an issue and get the rest of the changes in so the tests work again now that we have pywbem 0.15.0.  Remember that this PR is the basis for all subsequent PRs

I have no issue investigating this but want to separate that from getting the tests working again.  NOTE:Created issue #457 and marked that issue number with the two tests.

Reply:
ok